app/assets/stylesheets/control.scss

```diff
@@ -0,0 +1,19 @@
+.control {
+  color: #AAAAAA;
```

Comment:
Color literals like `#AAAAAA` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#AAAAAA` should be written as `#aaaaaa`